test/index.test.js

```diff
@@ -780,8 +780,19 @@ describe('Segment.io', function() {
         segment.options.crossDomainIdServers = [
           'userdata.example1.com',
           'xid.domain2.com',
-          'localhost'
```

Comment:
Is it OK to remove `localhost` as a test case, is there an equivalent test that includes this?